src/frontend/components/UI/WikiGameInfo/components/HowLongToBeat/index.scss

```diff
@@ -1,5 +1,7 @@
 .howLongToBeat {
```

Comment:
some change here broke the style in the old design
![image](https://github.com/Heroic-Games-Launcher/HeroicGamesLauncher/assets/188464/0e9185d2-8c5d-4b74-b8ac-e33c961b8a34)

Reply:
Fixed